lib/Settings/Settings.h

```diff
@@ -21,7 +21,7 @@
 
 #define WEB_INDEX_FILENAME "/web/index.html"
 
-#define MILIGHT_GITHUB_USER "sidoh"
+#define MILIGHT_GITHUB_USER "nothingmn"
```

Comment:
ack.  good catch.  already pushed up the change.

btw.  whats the easiest way to test out UI changes?